products/data_warehouse/backend/hogql_fixer_ai.py

```diff
@@ -248,7 +253,9 @@ def _parse_output(self, output, hogql_context: HogQLContext):
         # We also ensure the generated SQL is valid
         assert result.query is not None
         try:
-            print_ast(parse_select(result.query), context=hogql_context, dialect="clickhouse")
+            normalized_query = normalize_hogql_function_names(result.query)
+            result.query = normalized_query
+            print_ast(parse_select(normalized_query), context=hogql_context, dialect="clickhouse")
```

Comment:
Just running `parse_select(normalized_query)` here will get you the AST of the query to then use in my above comments